bin/mk-hscope.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-GHC_VERSION=8.8.3
-STREAMLY_VERSION=0.7.2
+GHC_VERSION=8.10.4
+STREAMLY_VERSION=0.8.0
```

Comment:
We can have a global environment for scripts in a separate shell file and source that file. Is it an overkill?

Reply:
I thought about that, but then there weren't too many places so ROI was low.